assets/data/themes/drauven.json

```diff
@@ -13,25 +13,13 @@
 			"canBeAwardedRandomly": false,
 			"slots": ["torso"],
 			"replaces": ["torso"],
-			"extraAspects": ["ATTACHMENTS_suppressArmor"],
+			"extraAspects": ["ATTACHMENTS_suppressArmor", "humanSkin_suppressClothes", "drauvenGear_clothes"],
 			"layers": [
 				{
 					"name": "Drauv_torso",
 					"tint": "skin", "tintAmount": 0.4,
 					"ifNoOwnerAspect": "themeSkin_drauven_alternate",
 					"depth": 1200
-				},
-				{
-					"name": "Drauv_armor_base",
-					"tint": "primary", "tintAmount": 0.6,
-					"ifNoOwnerAspect": "themeSkin_drauven_alternate",
-					"depth": 1350
-				},
-				{
-					"name": "Drauv_armor_base_over",
-					"tint": "primary", "tintAmount": 0.6,
-					"ifNoOwnerAspect": "themeSkin_drauven_alternate",
-					"depth": 1399
```

Comment:
You should leave in some clothing layers here so that games in progress don't suddenly have nekked Drauv